kapitan/cli.py

```diff
@@ -398,8 +400,34 @@ def secret_write(args, ref_controller):
         tag = '?{{ref:{}}}'.format(token_path)
         ref_controller[tag] = ref_obj
 
+    elif token_name.startswith("vault:"):
+        type_name, token_path = token_name.split(":")
+        _data = data.encode()
+        parameter = {}
+        encoding = 'original'
+        if args.base64:
+            _data = base64.b64encode(_data).decode()
+            _data = _data.encode()
+            encoding = 'base65'
```

Comment:
Fix the encoding scheme name.. base64 instead of base65.. Its surprising that despite setting the base65 encoding, the decoding is successful. This is potential bug scenario which needs to be fixed and new workflow test to be added as well.

Reply:
This wasn't detected since I removed the `vault` tests from `test_cli.py`. Also in unittest each component of the vault.py was tested. I will be adding the test for `cli` again after concluding the discussion regarding testing using docker container.